openpype/hosts/unreal/api/rendering.py

```diff
@@ -0,0 +1,125 @@
+import unreal
+
+from openpype.hosts.unreal.api import pipeline
+
+
+queue = None
+executor = None
+
+
+def _queue_finish_callback(exec, success):
+    unreal.log("Render completed. Success: " + str(success))
+
+    # Delete our reference so we don't keep it alive.
+    global executor
+    global queue
+    del executor
+    del queue
+
+
+def _job_finish_callback(job, success):
+    # You can make any edits you want to the editor world here, and the world
+    # will be duplicated when the next render happens. Make sure you undo your
+    # edits in OnQueueFinishedCallback if you don't want to leak state changes
+    # into the editor world.
+    unreal.log("Individual job completed.")
```

Comment:
Somehow I feel like we can avoid these globals to exist. E.g. we could define both these functions within the `start_rendering` function?